### ISSUE TITLE
Foam 0.13.5 Fails to Run

### ISSUE THREAD

**Issue by RemyPorter** [foambubble/foam#667]
- Foam version: 0.13.5
- Platform: Linux
- Issue occur on the [foam template](https://github.com/foambubble/foam-template) repo: Yes

**Summary**

This morning, VS Code automatically updated to 0.13.5, and Foam completely stopped working. It stopped showing up as a running extension. The details of my VS Code install are:

```
Version: 1.56.2
Commit: 054a9295330880ed74ceaedda236253b4f39a335
Date: 2021-05-12T16:45:26.313Z
Electron: 12.0.4
Chrome: 89.0.4389.114
Node.js: 14.16.0
V8: 8.9.255.24-electron.0
OS: Linux x64 5.4.0-74-generic snap
```

Reverting to 0.13.4 fixes the issue.

**Steps to reproduce**
1. Allow Foam to autoupdate to 0.13.5
2. Enjoy Foam crashing

**Additional information**

While there is no Foam entry in the output console, there is this crash report from the extension host:

```
- /snap/code/65/usr/share/code/resources/app/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:934:15)
	at Module._load (internal/modules/cjs/loader.js:779:27)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at v (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:694)
	at Object.<anonymous> (/home/remy/.vscode/extensions/foam.foam-vscode-0.13.5/node_modules/vfile/core.js:4:15)
	at Module.u._compile (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:1311)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
	at Module.load (internal/modules/cjs/loader.js:982:32)
	at Module._load (internal/modules/cjs/loader.js:823:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at v (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:694)
	at Object.<anonymous> (/home/remy/.vscode/extensions/foam.foam-vscode-0.13.5/node_modules/vfile/index.js:4:13)
	at Module.u._compile (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:1311)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
	at Module.load (internal/modules/cjs/loader.js:982:32)
	at Module._load (internal/modules/cjs/loader.js:823:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at v (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:694)
	at Object.<anonymous> (/home/remy/.vscode/extensions/foam.foam-vscode-0.13.5/node_modules/unified/index.js:8:13)
	at Module.u._compile (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:1311)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
	at Module.load (internal/modules/cjs/loader.js:982:32)
	at Module._load (internal/modules/cjs/loader.js:823:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at v (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:694)
	at Object.<anonymous> (/home/remy/.vscode/extensions/foam.foam-vscode-0.13.5/node_modules/foam-core/dist/foam-core.cjs.development.js:18:31)
	at Module.u._compile (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:1311)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
	at Module.load (internal/modules/cjs/loader.js:982:32)
	at Module._load (internal/modules/cjs/loader.js:823:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at v (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:694)
	at Object.<anonymous> (/home/remy/.vscode/extensions/foam.foam-vscode-0.13.5/node_modules/foam-core/dist/index.js:7:20)
	at Module.u._compile (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:1311)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
	at Module.load (internal/modules/cjs/loader.js:982:32)
	at Module._load (internal/modules/cjs/loader.js:823:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at v (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:694)
	at Object.<anonymous> (/home/remy/.vscode/extensions/foam.foam-vscode-0.13.5/out/extension.js:5:21)
	at Module.u._compile (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:4:1311)
	at Object.Module._extensions..js (internal/modules/cjs/loader.js:1155:10)
	at Module.load (internal/modules/cjs/loader.js:982:32)
	at Module._load (internal/modules/cjs/loader.js:823:14)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
	at Function.s._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:27816)
	at Function.S._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:24532)
	at Function.a._load (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:96:58822)
	at Module.require (internal/modules/cjs/loader.js:1006:19)
	at require (internal/modules/cjs/helpers.js:88:18)
	at Function.t [as __$__nodeRequire] (/snap/code/65/usr/share/code/resources/app/out/vs/loader.js:5:101)
	at d._loadCommonJSModule (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:106:25941)
	at d._doActivateExtension (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:92:12457)
	at d._activateExtension (/snap/code/65/usr/share/code/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:92:11516)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async Promise.all (index 0)
	at async Promise.all (index 89)
	at async Promise.all (index 1)
```

**Comment by riccardoferretti** [comment]
thanks for reporting this, I can reproduce it on my machine, even though I can't reproduce it when running the extension from code. Might be a packaging issue, will investigate

**Comment by riccardoferretti** [comment]
fixed in 0.13.7, which was just released

**Comment by RemyPorter** [comment]
Yep, that seems to do it. Awesome, thanks!
